File: javascript/src/models/otelTracingConfigOut.ts
// this file is @generated
import {
  type EndpointHeadersOut,
  EndpointHeadersOutSerializer,
} from "./endpointHeadersOut";

export interface OtelTracingConfigOut {
  url: string;
  headers: EndpointHeadersOut;
}

export const OtelTracingConfigOutSerializer = {
  _fromJsonObject(object: any): OtelTracingConfigOut {
    return {
      url: object["url"],
      headers: EndpointHeadersOutSerializer._fromJsonObject(object["headers"]),
    };
  },

  _toJsonObject(self: OtelTracingConfigOut): any {
    return {
      url: self.url,
      headers: EndpointHeadersOutSerializer._toJsonObject(self.headers),
    };
  },
};

File: rust/src/models/otel_tracing_config_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

use super::endpoint_headers_out::EndpointHeadersOut;

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct OtelTracingConfigOut {
    pub url: String,

    pub headers: EndpointHeadersOut,
}
